resources/leiningen/new/kraken/config.edn

```diff
@@ -1,3 +1,7 @@
 {:rabbit-mq {:connection {:host #resource-config/env "RABBITMQ_PORT_5672_TCP_ADDR"
                           :port #resource-config/edn #resource-config/env "RABBITMQ_PORT_5672_TCP_PORT"}
-             :queue "{{name}}"}}
+             :queues {"{{name}}.ok" {:exclusive false :auto-delete true}}}
+ :datomic {:uri "datomic:mem://{{name}}"
+           :partition :{{name}}
+           :intitialize true
```

Comment:
What if we replaces :initialize and :run-migrations as booleans with an :on-startup that takes a keyword? That might be cleaner?

`:on-startup :initialize`

```(condp = (config :datomic :on-startup)
  :initialize (db/initialize)
  :run-migrations (db/run-migrations)
  ...```